pbsmetrics/prometheus/prometheus.go

```diff
@@ -12,19 +12,20 @@ import (
 
 // Defines the actual Prometheus metrics we will be using. Satisfies interface MetricsEngine
 type Metrics struct {
-	Registry      *prometheus.Registry
-	connCounter   prometheus.Gauge
-	connError     *prometheus.CounterVec
-	imps          *prometheus.CounterVec
-	requests      *prometheus.CounterVec
-	reqTimer      *prometheus.HistogramVec
-	adaptRequests *prometheus.CounterVec
-	adaptTimer    *prometheus.HistogramVec
-	adaptBids     *prometheus.CounterVec
-	adaptPrices   *prometheus.HistogramVec
-	adaptErrors   *prometheus.CounterVec
-	cookieSync    prometheus.Counter
-	userID        *prometheus.CounterVec
+	Registry        *prometheus.Registry
+	connCounter     prometheus.Gauge
+	connError       *prometheus.CounterVec
+	imps            *prometheus.CounterVec
+	requests        *prometheus.CounterVec
+	reqTimer        *prometheus.HistogramVec
+	adaptRequests   *prometheus.CounterVec
+	adaptTimer      *prometheus.HistogramVec
+	adaptBids       *prometheus.CounterVec
+	adaptPrices     *prometheus.HistogramVec
+	adaptErrors     *prometheus.CounterVec
+	cookieSync      prometheus.Counter
+	adaptCookieSync *prometheus.CounterVec
```

Comment:
I'd personally be a bit confused by these names... in the code,  yes, but more importantly in the Prometheus metrics themselves.

In the dashboard you'd see `adapter_cookie_sync` and `usersync_total`, but the interesting difference between the two is that one logs `/cookie_sync` calls while the other logs `/setuid` stuff.

Maybe something like `cookie_sync_returns` and `setuid_attempts`? Or something else more clear.